### PR TITLE
[WIP] upgrade justinrainbow/json-schema dependancy to 2.0

### PR DESCRIPTION
**Pull request by mathroc**

### CODE REVIEW

**Comment by thewilkybarkid** [comment]
Ping @webmozart.

File: composer.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
     ],
     "require": {
         "php": "^5.3.3|^7.0",
-        "justinrainbow/json-schema": "^1.6",
+        "justinrainbow/json-schema": "^2.0",
```
Comment (xabbuh) [inline composer.json]: Is there a way to support both versions?

**Comment by mathroc** [comment]
Sorry, I'm not working on this anymore (I'm using justinrainbow/json-schema directly instead)

**Comment by thewilkybarkid** [comment]
No problem, I'll create a PR as it should be possible to support both.

**Pull request closed without merging**